src/pubkey.h

```diff
@@ -42,9 +42,11 @@ class CPubKey
      * script supports up to 75 for single byte push
      */
     static_assert(
-        PUBLIC_KEY_SIZE >= COMPRESSED_PUBLIC_KEY_SIZE,
-        "COMPRESSED_PUBLIC_KEY_SIZE is larger than PUBLIC_KEY_SIZE");
-
+        PUBLIC_KEY_SIZE > COMPRESSED_PUBLIC_KEY_SIZE,
+        "COMPRESSED_PUBLIC_KEY_SIZE should be smaller than PUBLIC_KEY_SIZE");
+    static_assert(
+        SIGNATURE_SIZE > COMPACT_SIGNATURE_SIZE,
```

Comment:
Is this a DER encoded signature vs. a 64kb (r,s)?  In this case this is not always correct as DER encoded signatures can be very small.